front-end/src/screens/Settings/email.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useContext, useEffect } from 'react'
+import { Icon } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+import Button from '../../ui-components/Button'
+import { Form } from '../../ui-components/Form'
+import { useChangeEmailMutation } from '../../generated/graphql'
+import { UserDetailsContext } from '../../context/UserDetailsContext'
+import { NotificationContext } from '../../context/NotificationContext'
+import { NotificationStatus } from '../../types'
+import { handleTokenChange } from '../../services/auth.service'
+
```

Comment:
We still didn't get that right:
`../../context/User` should be after `../../context/Notificatio`, those 2 should be before `../../generated/gr` .. etc.